src/mock_alchemy/mocking.py

```diff
@@ -468,7 +473,9 @@ class UnifiedAlchemyMagicMock(AlchemyMagicMock):
         "where": None,
     }
 
-    mutate: Set[str] = {"add", "add_all", "delete"}
+    mutate: Set[str] = {"add", "add_all", "delete", "execute"}
+
+    mutate_and_unify: Set[str] = {"execute"}
```

Comment:
[FMI] Why is mutate and unify? Shouldn't this be mutate or unify?

Also do we need `execute` to be part of mutate as well?

Reply:
Good point.
Execute should just be its own special case. Which can mutate data and potentially unify depending on what is being executed.